config/jenkins/configuration/clouds.yml

```diff
@@ -250,7 +250,7 @@ jenkins:
       cloudName: "AzureVMAgentsEastUs"
       configurationStatus: "pass"
       deploymentTimeout: 1200
-      newResourceGroupName: "<AZURE_VM_RESOURCE_GROUP>-westus"
```

Comment:
might want to postfix these with 2, as that is the region bein used to avoid.. the future problem

Reply:
What do you mean? Are you talking about West US 2?

Reply:
Yes, i am talking about west us 2, I believe that is the Azure deployment region being used.

Reply:
Changed below